src/GitHubClient/GitHubApi.cs

```diff
@@ -121,6 +152,12 @@ ex is TaskCanceledException
 
             foreach (T item in page.Nodes)
             {
+                if (excludedAuthors.Contains(item.Author.Login))
```

Comment:
That's the best I could come up with... I couldn't work out how to include the filter in the graphql query.

Reply:
We need to make sure this is case insensitive, but otherwise...

![MandoWayThisIsTheWayGIF](https://github.com/user-attachments/assets/806e7dac-19dc-4007-8941-328b3e416a1b)

Reply:
👍